app/views/cases/index.html.erb

```diff
@@ -6,22 +6,20 @@
       <th>Location</th>
       <th>Status</th>
       <th colspan="1"></th>
+    </tr>
   </thead>
-   </tr>
   <tbody>
     <% @cases.each do |case1| %>
```

Comment:
Move all this logic in to partial. File name should be `_cases.html.erb`. Please fix